### PR TITLE
Use consistent "Success" green for icon backgrounds

### PR DESCRIPTION
**Pull request by aduth**
**Why**: For consistency of our colors and available icons, and to avoid mismatching colors between badges.

Previously, there were three different colors used between badge borders and icon background:

- `#18852e` (Used for border color, also the standard `success` color token)
- `#63BB60` (Used for the "Unphishable" icon background)
- `#38bf53` (Used for the "Success" badge icon background)

With these changes, we use `#18852e` (`success` token) consistently.

**Screenshots:**

Icon|Before|After
---|---|---
Unphishable|Unphishable|Unphishable
Success|Success|Success

**Live preview:**

- Before: https://design.login.gov/components/badges/
- After: https://federalist-2f194a10-945e-4413-be01-46ca6dae5358.app.cloud.gov/preview/18f/identity-style-guide/aduth-icon-bg-success-color/components/badges/

### CODE REVIEW

**Comment by aduth** [comment]
Visual regression failure is, unsurprisingly, for the badge updates:

image

**Review by zachmargolis — APPROVED** [review]
LGTM

**Review by nickttng — APPROVED** [review]
LGTM. I've noted in LGDS Figma to update after this PR is merged.

File: src/img/alerts/unphishable.svg [hunk 1]
```diff
@@ -1,3 +1 @@
...
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" width="17" height="17"><path fill-rule="evenodd" d="M8.5 17a8.5 8.5 0 1 0 0-17 8.5 8.5 0 1 0 0 17zm4.25-8.652a.91.91 0 0 0-.911-.911h-.455V6.071A2.89 2.89 0 0 0 8.5 3.188a2.89 2.89 0 0 0-2.884 2.884v1.366h-.455a.91.91 0 0 0-.911.911v3.643a.91.91 0 0 0 .911.911h6.679a.91.91 0 0 0 .911-.911V8.348zM9.866 6.071v1.366H7.134V6.071A1.37 1.37 0 0 1 8.5 4.705a1.37 1.37 0 0 1 1.366 1.366zM7.65 9.623a.85.85 0 0 1 1.7 0v.971a.85.85 0 0 1-1.7 0v-.971z" fill="#18852e"/></svg>
```
[the remark this answers is not in the document]
Comment (aduth) [reply src/img/alerts/unphishable.svg:1]: This also reminds me: We should implement SVG optimization / validation in this project.

[the remark this answers is not in the document]
Comment (aduth) [reply src/img/alerts/unphishable.svg]: > This also reminds me: We should implement SVG optimization / validation in this project.

https://github.com/18F/identity-style-guide/pull/307

**Review by anniehirshman-gsa — COMMENTED** [review]
LGTMx3 thank you!

**Pull request merged**